papaparse.js

```diff
@@ -167,6 +168,42 @@
 		}
 	}
 
+	// Ability to parse file(s) when using with Node
+	function ParseFiles(_input, _config)
+	{
+		if (Array.isArray(_input)) {
+			var results = [];
+			_input.forEach(function(input) {
+				if(typeof input === 'object')
+					results.push(ParseFiles(input.file, input.config));
+				else
+					results.push(ParseFiles(input, _config));
+			});
+			return results;
+		} else {
+			var results = {
+				data: [],
+				errors: []
+			};
+			if ((/(\.csv|\.txt)$/).test(_input)) {
+				try {
+					var contents = fs.readFileSync(_input).toString();
```

Comment:
Yeah, readFileSync seems like the wrong call here (see http://stackoverflow.com/questions/13822085/fs-readfilesync-seems-faster-than-fs-readfile-is-it-ok-to-use-for-a-web-app-in for perspective from someone who does more Node than I do). I would suggest that you write a ChunkStreamer subclass (like FileStreamer) so you get the benefit of all of Papa's chunked reads and async processing.